src/App.tsx

```diff
@@ -11,7 +11,10 @@ import './board.scss';
 import './square.scss';
 import './player-score.scss';
 
-class App extends React.Component {
+interface AppProps {}
+interface AppState {}
+
+class App extends React.Component<AppProps, AppState> {
```

Comment:
This is just an example of how to type props and state when you have a class component